src/workerd/io/worker.h

```diff
@@ -168,6 +178,8 @@ class Worker: public kj::AtomicRefcounted {
   struct Impl;
   kj::Own<Impl> impl;
 
+	kj::HashMap<kj::String, ConnectFn> connectOverrides;
```

Comment:
Eek, it's a tab character!

Reply:
Nooooo another full CI wait time due to a tab 😢 I don't know how I managed to add that, but fixed.